maps/CEVEris/_CEV_Eris.dmm

```diff
@@ -1040,7 +1040,7 @@
 "atZ" = (/obj/random/powercell,/obj/random/powercell,/obj/random/material_rare/low_chance,/obj/random/boxes/low_chance,/obj/random/medical/low_chance,/obj/random/surgery_tool,/obj/random/circuitboard,/turf/simulated/floor/tiled/techmaint,/area/eris/maintenance/thirdsec5starboard)
 "aua" = (/obj/structure/cable/green{d1 = 1; d2 = 8; icon_state = "1-8"},/obj/machinery/hologram/holopad,/turf/simulated/floor/tiled,/area/eris/eschangarb)
 "aub" = (/turf/simulated/shuttle/wall/escpod{tag = "icon-escpodwall16"; icon_state = "escpodwall16"},/area/shuttle/escape_pod1/station)
-"auc" = (/obj/machinery/embedded_controller/radio/simple_docking_controller/escape_pod{frequency = 1380; id_tag = "escape_pod_1"; pixel_x = -25; pixel_y = 0; tag_door = "escape_pod_1_hatch"},/obj/structure/bed/chair/shuttle,/turf/simulated/shuttle/floor{icon_state = "cargofloor1"},/area/shuttle/escape_pod1/station)
+"auc" = (/obj/machinery/embedded_controller/radio/simple_docking_controller/escape_pod{frequency = 1380; id_tag = "escape_pod_1"; pixel_x = -25; pixel_y = 0; tag_door = "escape_pod_1_hatch"; tag = "escape_pod_1_controller"},/obj/structure/bed/chair/shuttle,/turf/simulated/shuttle/floor{icon_state = "cargofloor1"},/area/shuttle/escape_pod1/station)
```

Comment:
what have you done here and why?

Reply:
I've added tag to the shuttle docking controller, so pod could link itself with its controller. This was the cause of crash on initialization